public/app/features/explore/Explore.tsx

```diff
@@ -157,9 +157,9 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     this.onModifyQueries({ type: 'ADD_FILTER_OUT', options: { key, value } });
   };
 
-  onClickAddQueryRowButton = () => {
-    const { exploreId, queryKeys, datasourceInstance } = this.props;
-    this.props.addQueryRow(exploreId, queryKeys.length, datasourceInstance);
+  onClickAddQueryRowButton = async () => {
+    const { exploreId, queryKeys } = this.props;
+    await this.props.addQueryRow(exploreId, queryKeys.length);
```

Comment:
`await` does nothing here
```suggestion
  onClickAddQueryRowButton = () => {
    const { exploreId, queryKeys } = this.props;
    this.props.addQueryRow(exploreId, queryKeys.length);
```